lib/kube/proxy/forwarder.go

```diff
@@ -1502,9 +1503,10 @@ func (f *Forwarder) execNonInteractive(ctx *authContext, req *http.Request, _ ht
 		ConnectionMetadata:        connectionMetdata,
 		KubernetesClusterMetadata: ctx.eventClusterMeta(req),
 		KubernetesPodMetadata:     eventPodMeta,
-
-		InitialCommand:   request.cmd,
-		SessionRecording: ctx.recordingConfig.GetMode(),
+		InitialCommand:            request.cmd,
+		SessionRecording:          ctx.recordingConfig.GetMode(),
+		Invited:                   strings.Split(q.Get(teleport.KubeSessionInvitedQueryParam), ","),
```

Comment:
non-interactive sessions do not allow moderators to join so invited/reason shouldn't be present

Reply:
I think it's fine if we ignore them because we can't control what other people automations do so it will always be the case where they can append the info themselves